### ISSUE TITLE
`relayout!` sets/calls default template

### ISSUE THREAD

**Issue by Mo-Gul** [sglyon/PlotlyBase.jl#98]
When having a plot with a layout using a non-default template and then calling `relayout!` will set the default template.

Here an MWE

```julia
using PlotlyJS

data = scatter(x=1:5, y=1:5)
layout = Layout(template=:plotly_white)
p = Plot(scatter(x=1:5, y=1:5), layout)

relayout!(p, 
    xaxis=attr(
        linecolor=:red,
    )
)
```

results in

![image showing the result of above code](https://user-images.githubusercontent.com/17992976/171990085-0d7e2153-1075-4b16-83a2-b5523ca33d5b.png)

but should give

![image showing the _expected_ result of above code](https://user-images.githubusercontent.com/17992976/171990117-b1e3d200-6789-404b-9e2a-ff35bb4dc73f.png)

**Comment by hhaensel** [comment]
BTW, `template = :plotly_white` doesn't really work except that by chance it returns a light layout. These definitions are not part of plotlyjs but of the [python plotlyjs library](https://github.com/plotly/plotly.py/blob/c54a2bdf1655caaaa3b7b71fbfc38a5584767bd5/templategen/definitions.py#L423)

**Comment by hhaensel** [comment]
Feel free to contribute respective Julia themes. It shouldn't be too hard.

**Comment by hhaensel** [comment]
So the correct call would be
```julia
Plot(scatter(y = 1:5), Layout(template = templates[:plotly_dark]))
```

**Comment by Mo-Gul** [comment]
@hhaensel, sure that `template=:plotly_white` shouldn't work? Because so far it does and does seem to do the same as `template=templates[:plotly_white]`...

I'll test again when the new version is out if something has changed.

**Comment by hhaensel** [comment]
Please test :plotly_dark

**Comment by Mo-Gul** [comment]
You are right, when I use `:plotly_dark` it also results in "white". But the same happens when I use `"bla"` or `555`. It would be nice if (at some point) a warning or error would be thrown that this template wasn't found and ... was used instead.

If you consider this a bug (as well), shall I open an issue or do you want to do it yourself?
